MultiServer.py

```diff
@@ -1942,9 +1942,14 @@ async def process_client_cmd(ctx: Context, client: Client, args: dict):
             ctx.save()
             ctx.on_changed_hints(client.team, hint.finding_player)
             ctx.on_changed_hints(client.team, hint.receiving_player)
-        
+
         elif cmd == 'StatusUpdate':
-            update_client_status(ctx, client, args["status"])
+            if args["status"] == ClientStatus.CLIENT_GOAL and client.no_locations:
+                await ctx.send_msgs(client, [{'cmd': 'InvalidPacket', "type": "cmd",
+                                              "text": "Trackers can't register Goal Comple",
```

Comment:
got distracted in the middle of typing a word

Reply:
```suggestion
                                              "text": "Trackers can't register Goal Complete",
```
I appreciate the humor but I literally could not find what you were trying to comment on for a bit, lol